seatingchart.py

```diff
@@ -68,15 +68,18 @@ def working_dir_path(name, slug, extension):
 ASSIGN_LAST  = assign_last_path  if os.path.isfile(assign_last_path)  else "/dev/null"
 
 # 3) a HTML page matching the courseworks print as single column roster feature, with student pictures
-roster_path = working_dir_path("roster", args.slug, "html")
-assert_file_exists(roster_path)
-ROSTER_PAGE = roster_path
+# roster_path = working_dir_path("roster", args.slug, "html")
+# assert_file_exists(roster_path)
+# ROSTER_PAGE = roster_path
```

Comment:
we can just delete these lines and get it back from the git history if it's actually needed